cdist/exec/remote.py

```diff
@@ -140,8 +140,7 @@ def extract_archive(self, path, mode):
         # AIX only allows -C at the end
         command = [
             "tar",
-            "-x",
-            "-f", path,
+            "xf", path,
```

Comment:
I cannot comment on it, because it's outside of the diff, but I think we'd have to change `ArchivingMode.extract_opts`, too, to match the `-`-less form.